test/ngRoute/directive/ngViewSpec.js

```diff
@@ -913,9 +912,11 @@ describe('ngView animations', function() {
 
       $location.path('/foo');
       $rootScope.$digest();
-      expect($animate.queue.shift().event).toBe('enter');
-      $animate.triggerCallbacks();
 
+      $animate.flush();
+      $rootScope.$digest();
```

Comment:
why do we need the extra digest after flush? it wasn't necessary before and we digest just before flushing already.